services/core-api/app/api/verify/permit/mine/resources/verify_permit_mine_resource.py

```diff
@@ -0,0 +1,73 @@
+from datetime import datetime
+
+from flask_restplus import Resource, fields
+from flask import request
+from app.extensions import api
+
+from app.api.mines.mine.models.mine import Mine
+from app.api.mines.permits.permit.models.permit import Permit
+
+from app.api.utils.access_decorators import requires_role_view_all
+
+VERIFY_PERMIT_MINE_MODEL = api.model(
+    'VerifyPermitMineModel', {
+        'a_Result': fields.String,
+        'a_MineInfo': fields.String,
+        'a_ResponseMessage': fields.String,
+        'a_Timestamp': fields.DateTime
+    })
+
+class VerifyPermitMineResource(Resource):
+    @api.doc(
+        description=
+        'Verifies that a permit is valid for a mine based on the type of deemed authorization. NOTE: This exists for integration purposes and does not follow the typical patterns of this API.',
+        params={
+            'a_PermitNumber': f'The permit number.',
+            'a_TypeofDeemedAuth': f'The type of deemed authorization, eg INDUCED.'
+        }
+    )
+    @api.marshal_with(VERIFY_PERMIT_MINE_MODEL, code=200)
+    @requires_role_view_all
+    def get(self):
+        result = ""
+        mine_info = ""
+        response_message = ""
+
+        try:
+            permit_no = request.args.get('a_PermitNumber')
+            permit_prefix = permit_no.split("-")[0]
+            type_of_deemed_auth = request.args.get('a_TypeofDeemedAuth')
+
+            permits = Permit.find_by_permit_no_all(permit_no)
```

Comment:
Since this is constrained to just the `permit_no` I don't think its an issue, but if it were bigger I'd suggest using ORM filters below, rather than iterating in memory.